Composer/packages/client/src/pages/botProject/runtime-settings/RuntimeSettings.tsx

```diff
@@ -95,6 +102,12 @@ export const RuntimeSettings: React.FC<RouteComponentProps<{ projectId: string }
 
     setRuntimeField(projectId, field, newValue);
 
+    if (field === RuntimeType.Path) {
+      setRuntimePath(newValue);
+    } else {
+      setRuntimeCommand(newValue);
+    }
```

Comment:
I recommend that take the local state setters out side because when the onChange handler is triggered in the input box, it already knows where should the local state go.

Suggestion:
```jsx
// It doesn't return a function
const updateSetting = (field, val) => {
  // Error handling logic
  setRuntimeField(projectId, fieldId, val);
}

<TextField
  id="runtime-command"
  onChange={(e, val) => { // Can define this handler outside JSX
    setRuntimeCommand(val);
    updateSettings(val, 'command');
  }}}
/>
```